internal/tiltfile/tiltfile_state.go

```diff
@@ -911,7 +924,8 @@ func (s *tiltfileState) translateK8s(resources []*k8sResource) ([]model.Manifest
 
 		m = m.WithDeployTarget(k8sTarget)
 
-		iTargets, err := s.imgTargetsForDependencyIDs(r.dependencyIDs)
+		registry := s.registry()
```

Comment:
can you move this out of the resources loop?

Reply:
damnit -_- ha yes will do